db/migrate/20220519225648_create_roles.rb

```diff
@@ -0,0 +1,10 @@
+class CreateRoles < ActiveRecord::Migration[6.1]
+  def change
+    create_table :roles do |t|
+      t.string :name
```

Comment:
Es una buena práctica agregar la validación tanto en el modelo como en el base de datos. Por ejemplo:
```suggestion
      t.string :name, null: false
```
Que se correspondería con el `validates :name, presence: true`